DiagnosticsExtension/DiagnosticsExtension.csproj

```diff
@@ -56,14 +56,44 @@
     <Reference Include="Antlr3.Runtime, Version=3.5.0.2, Culture=neutral, PublicKeyToken=eb42632606e9261f, processorArchitecture=MSIL">
       <HintPath>..\packages\Antlr.3.5.0.2\lib\Antlr3.Runtime.dll</HintPath>
     </Reference>
+    <Reference Include="Azure.Core, Version=1.21.0.0, Culture=neutral, PublicKeyToken=92742159e12e44c8, processorArchitecture=MSIL">
```

Comment:
Could you please do a review of packages being included here to see if any were included during development (trial and error) and not actually needed?  This seems like a large number of new packages and its possible that all are needed but want to be sure we are not bloating the solution unnecessarily.